osu.Game/Graphics/Containers/SectionsContainer.cs

```diff
@@ -35,7 +35,7 @@ public Drawable ExpandableHeader
                 if (value == expandableHeader) return;
 
                 if (expandableHeader != null)
-                    RemoveInternal(expandableHeader);
+                    RemoveInternal(expandableHeader, false);
```

Comment:
Any special reason for this one to be `false` when the one below is `true`?

Reply:
Was an oversight, fixed.